clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -86,136 +89,134 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
-            <DescriptionList>
-              {storageFields?.s3Fields && (
-                <>
-                  <DashboardDescriptionListGroup
-                    title="Endpoint"
-                    isEmpty={!storageFields.s3Fields.endpoint}
-                    contentWhenEmpty="No endpoint"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-endpoint"
-                      textToCopy={storageFields.s3Fields.endpoint}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Region"
-                    isEmpty={!storageFields.s3Fields.region}
-                    contentWhenEmpty="No region"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-region"
-                      textToCopy={storageFields.s3Fields.region || ''}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Bucket"
-                    isEmpty={!storageFields.s3Fields.bucket}
-                    contentWhenEmpty="No bucket"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-bucket"
-                      textToCopy={storageFields.s3Fields.bucket}
-                    />
-                  </DashboardDescriptionListGroup>
-                  <DashboardDescriptionListGroup
-                    title="Path"
-                    isEmpty={!storageFields.s3Fields.path}
-                    contentWhenEmpty="No path"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-path"
-                      textToCopy={storageFields.s3Fields.path}
-                    />
-                  </DashboardDescriptionListGroup>
-                </>
-              )}
-              {(storageFields?.uri || storageFields?.ociUri) && (
-                <>
-                  <DashboardDescriptionListGroup
-                    title="URI"
-                    isEmpty={!modelArtifact?.uri}
-                    contentWhenEmpty="No URI"
-                  >
-                    <InlineTruncatedClipboardCopy
-                      testId="storage-uri"
-                      textToCopy={modelArtifact?.uri || ''}
-                    />
-                  </DashboardDescriptionListGroup>
-                </>
-              )}
+    <Card>
+      <CardHeader>
+        <Title headingLevel="h2">Version details</Title>
+      </CardHeader>
+      <CardBody>
+        <Flex
+          direction={{ default: 'column', md: 'row' }}
+          columnGap={{ default: 'columnGap4xl' }}
+          rowGap={{ default: 'rowGapLg' }}
+        >
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <DescriptionList isFillColumns>
+              <EditableLabelsDescriptionListGroup
```

Comment:
Please reuse https://github.com/kubeflow/model-registry/blob/main/clients/ui/frontend/src/app/pages/modelRegistry/screens/components/ModelEditableLabelsDescriptionListGroup.tsx here, you can change the props to pass into `labels`, `onLabelChange`, `allExistingKeys` and so on as needed.